resources/views/admin/auth-clients/index.blade.php

```diff
@@ -8,11 +8,13 @@
     @include('layouts.sidebar', ['sidebar'=> Menu::get('sidebar_admin')])
 @endsection
 
-@section('content')
+@section('breadcrumbs')
```

Comment:
breadcrumbs section should be defined in the layout.blade not in the individual blades

Reply:
@yield('breadcrumbs') is already defined in the layout.blade and was used in modeler, screen builder, script editor, ...